frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterSet, FilterValue, RangeFilterValue } from "../../models/paged_items";
+
+interface FiltersInfosProps {
+    filters: FilterSet
+}
+
+const FiltersInfos = ({ filters }: FiltersInfosProps) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
+    const getValue = (key: any) => {
+        const filterValue: FilterValue = filters[key].value ?? {}
+        const description: any = filters[key].description
+        let valueJSX: any;
+        const descriptionJSX: any = (
```

Comment:
You can leave the type off here, as it can be inferred by the compiler to be JSX.Element.